Automattic/wp-calypso
# Jetpack connect: store url in redux

Since all the redirections and routing of jetpack connect now happens in calypso and not in the 3rd party site, we don't need to store urls & session info in the user settings anymore. 

This PR change how we store the entered urls to use the persistent redux store instead of sending them to the API to store them in the database.

How to test
========

1. You will need to have the redux extension of the chrome dev tools to check if the url have been stored
2. Go to http://calypso.localhost:3000/jetpack/connect and enter a valid url
3. In the authorize screen (or any other), you should be able to see the url and a timestamp stored in state/jetpackConnect/jetpackConnectSessions

ping @roccotripaldi @ryelle
